src/components/MaterialUI/DataDisplay/SelectionList.js

```diff
@@ -80,7 +88,12 @@ const SelectionList = ({
 					onChange={onChangeEvent}
 				/>
 			</Grid>
-			<Grid item xs={7}>
+			{infoPanel && (
+				<Grid item xs={1}>
+					<Divider dividerProps={dividerProps} />
+				</Grid>
+			)}
+			<Grid item xs={6}>
```

Comment:
should we have a condition if there is no infoPanel that this Grid item width is 7 instead of 6 ?

Reply:
Makes sense, added it...  will push changes shortly.